### ISSUE TITLE
[Bug Report][3.11.2] Odd behavior of v-select when having multiple items with same prefix

### ISSUE THREAD

**Issue by tonqa** [vuetifyjs/vuetify#22423]
### Environment
**Vuetify Version:** 3.11.2
**Vue Version:** 3.5.25
**OS:** Windows 10 (current)

### Steps to reproduce
1. Use a v-select with "ids", "ids1", "ids2", "ids3" as items.
2. Open the browser and select the v-select component with the mouse.
3. Type "ids", which selects the THIRD item "ids2" instead of "ids".

### Expected Behavior
When typing "ids" in the keyboard, the v-select selects the FIRST item which has "ids" as prefix.

### Actual Behavior
When typing "ids" in the keyboard, the v-select selects the THIRD item, which has "ids" as prefix, which is odd.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVjz0OgzAMha8SeWFJgwCxoBSpZ+gIDCl1JaQ0jUjKUvXuza8o07O/J9vPwwfMOpcXrdn2RuiAW3xqKSz2oyKEbyehdShDM7+UFYvCNaEADUqcbQaESHFDeR7hGjnsTre47cZZQ7HcTUGJlyppnbQppn2m52U+kFM4cIzhSUzJy7/08KXQsJbVLfiiqlgN9CGkQXr4ObLpB7BzT0U=)

### Other comments
This behaviour is really odd and should be fixed. We use many items with the same prefix and don't accept this behaviour and urgently wait for a fix.

**Comment by tonqa** [comment]
A greatful thank you for handling this quick. This was released in just 15 minutes. 👍
